app/controllers/observations_controller/form_helpers.rb

```diff
@@ -84,20 +84,25 @@ def init_project_vars
     @projects = User.current.projects_member(order: :title,
                                              include: :user_group)
     @project_checks = {}
-    @projects.each do |proj|
-      @project_checks[proj.id] = (proj.open_membership &&
-                                  proj.accepting_observations)
-    end
+
+    # FIXME: Why wouold open-membership projects be treated different from
+    # closed-membership projects for initial auto-checking?
+    # And why auto-check non-compliant projects?
+    # @projects.each do |proj|
+    #   @project_checks[proj.id] = proj.open_membership
+    # end
+    # 2023-10-05 jdc
```

Comment:
@mo-nathan: 
Please see the FIXME.
Also note that I changed functionality:
No projects are checked when a user creates an Observation when there's no recently-created Observation.
I didn't understand why any projects wold be checked in that scenario.
I also didn't understand why open- and closed- projects would get different treatment.
If you feel that open-membership projects should be checked in this scenario, I'll fix things up.
I'll need to redo some tests. I should probably also insure that projectgs are not auto-check if constrains are violated.

Reply:
@JoeCohen My rational is that if a user joins a project and it is currently active and they create a new observation it should by default go into that project.  Imagine someone at a NAMA foray.  They join a project and then immediately make an observation.  They pretty obviously want that observation to be in that project regardless of what they last chose.  Projects with closed membership are a bit different since in my view since there has to be some dialog there and I don't necessarily expect that all observations will be wanted there.  It's also been the historic behavior of those projects, so I didn't think I should change that.  However, I feel more strongly that folks joining open projects should have that project on by default if the constraints are met than I do about the current behavior of closed projects.

Reply:
Thanks for the explanation. I'll make open projects auto-checked if constrains are met.